<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemWnd</class>
 <widget class="QMainWindow" name="MemWnd">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>667</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MemWnd</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>696</width>
      <height>560</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hrzAll">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Instructions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listView">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>400</height>
          </size>
         </property>
         <property name="autoScroll">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Screen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="textBrowser">
             <property name="font">
              <font>
               <family>Courier New</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::NoTextInteraction</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="grpRegisters">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>170</height>
              </size>
             </property>
             <property name="title">
              <string>Registers</string>
             </property>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>21</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>AX:</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="txtAX">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>20</y>
                <width>41</width>
                <height>20</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Courier New</family>
               </font>
              </property>
              <property name="text">
               <string>0000</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>50</y>
                <width>21</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>BX:</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="txtBX">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>50</y>
                <width>41</width>
                <height>20</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Courier New</family>
               </font>
              </property>
              <property name="text">
               <string>0000</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>80</y>
                <width>21</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>CX:</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="txtCX">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>80</y>
                <width>41</width>
                <height>20</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Courier New</family>
               </font>
              </property>
              <property name="text">
               <string>0000</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>110</y>
                <width>21</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>DX:</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="txtDX">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>110</y>
                <width>41</width>
                <height>20</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Courier New</family>
               </font>
              </property>
              <property name="text">
               <string>0000</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>140</y>
                <width>21</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>SI:</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="txtSI">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>140</y>
                <width>41</width>
                <height>20</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Courier New</family>
               </font>
              </property>
              <property name="text">
               <string>0000</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_7">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>20</y>
                <width>21</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>SP:</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="txtSP">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>20</y>
                <width>41</width>
                <height>20</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Courier New</family>
               </font>
              </property>
              <property name="text">
               <string>0000</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_8">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>50</y>
                <width>21</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>BP:</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="txtBP">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>50</y>
                <width>41</width>
                <height>20</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Courier New</family>
               </font>
              </property>
              <property name="text">
               <string>0000</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_9">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>80</y>
                <width>21</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>IP:</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="txtIP">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>80</y>
                <width>41</width>
                <height>20</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Courier New</family>
               </font>
              </property>
              <property name="text">
               <string>0000</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="txtFLAGS">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>110</y>
                <width>41</width>
                <height>20</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Courier New</family>
               </font>
              </property>
              <property name="text">
               <string>0000</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_10">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>110</y>
                <width>21</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>FG:</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="txtDI">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>140</y>
                <width>41</width>
                <height>20</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Courier New</family>
               </font>
              </property>
              <property name="text">
               <string>0000</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_11">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>140</y>
                <width>21</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>DI:</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="grpFlags">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>120</height>
              </size>
             </property>
             <property name="title">
              <string>Flags</string>
             </property>
             <widget class="QCheckBox" name="chkCarry">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>71</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Carry</string>
              </property>
             </widget>
             <widget class="QCheckBox" name="chkOverflow">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>30</y>
                <width>71</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Overflow</string>
              </property>
             </widget>
             <widget class="QCheckBox" name="chkZero">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>60</y>
                <width>71</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Zero</string>
              </property>
             </widget>
             <widget class="QCheckBox" name="chkSign">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>60</y>
                <width>71</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Sign</string>
              </property>
             </widget>
             <widget class="QCheckBox" name="chkParity">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>90</y>
                <width>71</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Parity</string>
              </property>
             </widget>
             <widget class="QCheckBox" name="chkAdjust">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>90</y>
                <width>71</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Adjust</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>250</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Load_Object_File"/>
   <addaction name="actionRun"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_Debug">
    <property name="title">
     <string>&amp;Debug</string>
    </property>
    <addaction name="action_Run"/>
    <addaction name="action_Stop"/>
    <addaction name="separator"/>
    <addaction name="actionStep_Into"/>
    <addaction name="actionStep_Over"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Debug"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Load_Object_File">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/res/open.ico</normaloff>:/res/open.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Object File</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/res/run.ico</normaloff>:/res/run.ico</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/res/open.ico</normaloff>:/res/open.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Object File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Run">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/res/run.ico</normaloff>:/res/run.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStep_Into">
   <property name="text">
    <string>Step &amp;Into</string>
   </property>
  </action>
  <action name="actionStep_Over">
   <property name="text">
    <string>Step &amp;Over</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
